main.go

```diff
@@ -58,11 +62,50 @@ func listAll(w http.ResponseWriter, req *http.Request) {
     w.Write([]byte(result))
 }
 
+func upload(w http.ResponseWriter, req *http.Request) {
+	req.ParseMultipartForm(10 << 30)
```

Comment:
What's this?

Reply:
It's to parse body file with size limit. I can remove it actually since it's not too important